File: assets/scripts/constantrotation.py
import dengine.common as common
import dengine.scene as scene
import dengine.timer as timer


speed = 4.0
def update(entity):
    delta = timer.get_delta()
    delta_s = delta / 1000.0
    entity.transform.rotation.y += delta_s * speed
    
    

File: nsl/src/nsl-test/constantrotation.c
#include <dengine/dengine.h>

int constantrotation_update(Entity* entity)
{
    static const double speed = 4.0;
    double delta = dengineutils_timer_get_delta();
    double delta_s = delta / 1000.0;
    entity->transform.rotation[1] += delta_s * speed;

    return 0;
}
